app/templates/community/invasion-of-ukraine.hbs

```diff
@@ -0,0 +1,130 @@
+        </li>
+        <li class="text-hero-base mb-2">
+          Russian troops launched missile strikes against peaceful Ukrainian cities from various directions, including
+          the temporarily occupied Donbas and Crimea and the northeastern region.
+        </li>
+        <li class="text-hero-base mb-2">
+          Russia has struck military and strategically critical infrastructure all over the country.
+          Borders are violated from the north, east, and the administrative border with Crimea.
+          The Ukrainian Armed Forces are retaliating; the enemy has casualties.
+        </li>
+        <li class="text-hero-base mb-2">
+          This is an act of war, an attack on Ukraine's sovereignty and territorial integrity, a gross violation of the
+          UN Charter and the fundamental norms and principles of international law.
+          Accordingly, Ukraine has activated its right to self-defense under international law.
+        </li>
+        <li class="text-hero-base mb-2">
+          The situation is challenging. Russia's primary goal is to destabilize our state, destroy institutions and
+          raise panic in society.
+        </li>
+        <li class="text-hero-base mb-2">
+          The Ukrainian people ask the world to respond as harshly as possible to what is happening. Ukraine calls on
+          international partners to immediately apply a new round of meaningful sanctions, against the aggressor.
+        </li>
+      </ul>
+
+      <p class="text-hero-base mb-2">
+        It’s exactly the WAR, not “military operation”, as Putin said.
+        Ukraine needs help of foreign governments in the form of intrusion of military contingent
+      </p>
+
+      <div class="inner">
+        <h2>
+          HOW YOU CAN SUPPORT UKRAINE
+        </h2>
+
+        <p>
+          Follow and share information about the real situation in Ukraine among your friends (<a
+          href="https://kyivindependent.com">Kyiv Independent</a>; <a href="https://www.instagram.com/svidomi_eng">Svidomi
+          (en)</a>).
+        </p>
+
+        <p>
+          Join the rallies against the Russian Federation in your city!
+          Here you can see the schedule of the rallies <a href="https://www.stopputin.net">Stop Putin</a>
+        </p>
+
+        <h3>
+          Support Ukrainian Army
+        </h3>
+        <ul>
+          <li class="text-hero-base mb-2">
+            <p>
+              🇺🇦 The National Bank of Ukraine has opened a <a
+              href="https://bank.gov.ua/en/news/all/natsionalniy-bank-vidkriv-spetsrahunok-dlya-zboru-koshtiv-na-potrebi-armiyi">special
+              account to raise funds for the Ukrainian Army</a>.
+              Anyone from any country can donate!
+            </p>
+            <p>
+              The account is multi-currency. It is opened for transfers of funds from international partners and donors,
+              any person as well as from Ukrainian business and citizens.
```

Comment:
```suggestion
              The account is multi-currency. It can receive fund transfers from international partners, donors, and Ukrainian businesses and citizens.
```